spec/views/guide/_content_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe 'guide/_content', type: :view do
+  it 'links to the structures node' do
+    render
+    expect(rendered).to match(%r{<a class="[^"]+" href="/guide/structures">})
```

Comment:
In the test suite, `guide` is mounted at `/guide`. This test proves the link is rendered with account for the engine mount point.